node/src/main/scala/coop/rchain/node/diagnostics/client/Runtime.scala

```diff
@@ -16,8 +16,10 @@ object Runtime {
 
   def diagnosticsProgram[F[_]: Monad: ErrorHandler: ConsoleIO: DiagnosticsService]: F[Unit] = {
     val program = for {
-      peers   <- DiagnosticsService[F].listPeers
-      _       <- ConsoleIO[F].println(showPeers(peers))
+      peersRP <- DiagnosticsService[F].listPeers
```

Comment:
it's calling same service twice, this you probably want to call `listDiscoveredPeers` here right?